recipes/phytop/src/small_tools.py

```
@@ -0,0 +1,358 @@
+	return os.path.exists(xfile) and os.path.getsize(xfile)>0
+def test_r(xfile): # readable
+	try: 
+		f = open(xfile)
+		f.close()
+		return True
+	except:
+		return False
+def is_complete_cds(Seq, translate_table=1):
+	'''A Bio.Seq object (CDS) is complete (both start and end codons present)?'''
+	try:
+		pro_seq = Seq.translate(table=translate_table, to_stop=True, cds=True)
```

Ruff

Local variable `pro_seq` is assigned to but never used

Remove assignment to unused variable `pro_seq`

(F841)